website/docs/reference/resource-properties/constraints.md

```diff
@@ -303,6 +303,67 @@ select
 
 </File>
 
+### Custom constraints on models for advanced configuration of tables
+
+In dbt Cloud, you can use custom constraints on models for advanced configuration of tables. Custom constraints allow you to set [masking policies](https://docs.snowflake.com/en/user-guide/security-column-intro#what-are-masking-policies) in Snowflake when using a  Create Table As Select (CTAS).
+
+Masking policies conceal sensitive data from unauthorized access, while still allowing authorized users to retrieve the data during query execution.
+
+Contracts and constraints support tag-based masking policies using the following syntax:
```

Comment:
i think before introducing the tag way, should we say there are a couple of ways to do this:

- the tag way (with a link to the tag way)
- the non-tag way (with a link to that)